parachain/pallets/control/src/lib.rs

```diff
@@ -299,6 +356,27 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// - `update`: (ControlOperation, Fee).
+		#[pallet::call_index(6)]
+		#[pallet::weight(T::WeightInfo::update_operation_fee())]
+		pub fn update_operation_fee(
```

Comment:
I think these fees will be updated very rarely. By making them static config, we would not need to maintain and audit this `update_operation_fee` extrinsic.

Reply:
For example, the Pallet config, can have a `Get<BTreeMap<ControlOperation, BalanceOf<T>>` which contains the fees.

https://paritytech.github.io/substrate/master/sp_std/collections/btree_map/struct.BTreeMap.html

Reply:
> I think these fees will be updated very rarely. By making them static config, we would not need to maintain and audit this update_operation_fee extrinsic.

I would assume the fee charged here is supposed to cover the cost in ethereum side, considering the volatile gas price and fee model so I would prefer to make it configurable and not a static config.

Reply:
ok good point! Yes, I realise the fee charged in the pallet needs to cover the reward given to the relayer on Ethereum side (from funds in BridgeHub's agent contract)